common/static_modifiers/hfe_static_modifiers.txt

```diff
@@ -107,10 +107,44 @@ hfe_fallen_resources_drain = {
 	pop_government_ethic_attraction = -0.3
 }
 
+hfe_harvesting_marvel_unkeep = {
+	country_starbase_influence_cost_mult = 2
+	country_naval_cap_mult = 1
+	country_resource_max_alloys_add = 99999
+	country_resource_max_minerals_add = 99999
+}
+
+hfe_harvesting_marvel_boost = {
+	country_energy_produces_add = 30 # This is the default for Vanilla rendition empires as well
+	country_minerals_produces_add = 300
```

Comment:
This is to prevent an AI death spiral, as well as to provide flavor. It's a Harvesting MARVEL for a reason. The AI still has a maximum it can hold, and the Harvesting Marvel can use the Galactic Market.